v0/destinations/yahoo_dsp/transform.js

```diff
@@ -0,0 +1,201 @@
+const {
+  BASE_ENDPOINT,
+  ENDPOINTS,
+  DSP_SUPPORTED_OPERATION,
+  AUDIENCE_TYPE
+} = require("./config");
+const {
+  defaultRequestConfig,
+  CustomError,
+  defaultPutRequestConfig,
+  removeUndefinedAndNullValues,
+  getSuccessRespEvents,
+  getErrorRespEvents
+} = require("../../util");
+
+const {
+  getAccessToken,
+  populateIncludes,
+  populateExcludes,
+  createPayload
+} = require("./util");
+
+/**
+ * This function is used for building the final response to be returned.
+ * @param {*} message
+ * @param {*} destination
+ * @returns
+ */
+const responseBuilder = async (message, destination) => {
+  let dspListPayload = {};
+  const { Config } = destination;
+  const { listData } = message.properties;
+  const { accountId, audienceId, audienceType, seedListType } = Config;
+
+  let domains = [];
+  let categoryIds = [];
```

Comment:
use const for `domains` and `categories` as they are never reassigned.

Reply:
done